src/index.ts

```diff
@@ -108,54 +116,102 @@ function createPreprocessor(
 	let stopped = false;
 	emitter.on("exit", done => {
 		stopped = true;
-		bundle.stop().then(done);
+		bundlerMap.stop().then(done);
+	});
+
+	// Logging
+	const pendingBundles = new Set<string>();
+	let start = 0;
+	bundlerMap.on("start", ev => {
+		if (singleBundle) {
+			start = ev.time;
+			log.info(`Compiling to ${ev.file}...`);
+		} else if (!pendingBundles.size) {
+			start = ev.time;
+			log.info(`Compiling...`);
+		}
+		pendingBundles.add(ev.file);
+	});
+
+	bundlerMap.on("stop", ev => {
+		pendingBundles.delete(ev.file);
```

Comment:
I think we can just `clear` here? Stopping stops everything. We might even need this event, the karma process will die shortly.